packages/app-desktop/gui/lib/SearchInput/SearchInput.tsx

```diff
@@ -33,6 +33,10 @@ export const SearchInput = styled(StyledInput)`
 	padding-right: 20px;
 	flex: 1;
 	width: 10px;
+
+	&::-webkit-search-cancel-button {
```

Comment:
See https://stackoverflow.com/a/58484957 for how to inspect and select other elements in the Electron-provided search element.